source/class/qx/ui/form/AbstractField.js

```diff
@@ -125,9 +125,18 @@ qx.Class.define("qx.ui.form.AbstractField", {
     if (value != null) {
       this.setValue(value);
     }
-
-    this.getContentElement().addListener("change", this._onChangeContent, this);
-
+    let el = this.getContentElement();
+    // change does not fire on ios webkit when just the keyboard is closed
+    // blur does fire though ... 
+    // since ios allows no other html engines, checking for ios should do
+    if (qx.core.Environment.get("os.name") == "ios") {
+      el.addListener("blur", () => {
+        this._onChangeContent(new qx.event.type.Data(this.getValue()));
```

Comment:
You should only fire this if the valkue has really changed!

Reply:
yes I agree, except that in this case, code further down is already taking care of this ...